app/bundles/LeadBundle/Helper/TokenHelper.php

```diff
@@ -43,6 +44,17 @@ public static function findLeadTokens($content, $lead, $replace = false)
                 $tokenList[$token] = self::getTokenValue($lead, $alias, $defaultValue);
             }
 
+            foreach ($dateMatches[2] as $key => $match) {
+                $token = $dateMatches[0][$key];
+
+                if (isset($tokenList[$token])) {
+                    continue;
+                }
+
+                $dt                = new DateTimeHelper($match);
+                $tokenList[$token] = $dt->toLocalString('Y-m-d H:i:s');
```

Comment:
Please use this constant instead of repeating the string throughout this PR.
```suggestion
                $tokenList[$token] = $dt->toLocalString(DateTimeHelper::FORMAT_DB);
```